Modify the sqlite setup instructions for Windows

iglpdc opened the pull request:
This is taken care by the Windows installer and the previous link to our files
page was broken. This fixes #192.

iglpdc committed:
Modify the sqlite setup instructions for Windows.

This is taken care by the Windows installer and the previous link to our files
page was broken.

fmichonneau commented:
Hi @iglpdc, Thanks for this. It looks good. I'm a little worried that some people might get confused as we now ask to run the windows installer multiple times in the instructions (this is relatively independent from your pull request). Could you maybe add a sentence to the effect of: "If you have already used the Windows installer previously, you don't need to run it again."

iglpdc committed:
Clarify that the SWC Windows installer runs once.

The SWC installer need to be run only once, so if you used it when setting up
nano, you don't need to run it again to install sqlite3.

iglpdc commented:
@fmichonneau I've added a comment saying that you don't need to run the installer once.

fmichonneau commented:
That looks good to me. We'll use #194 to keep track of the more general issue.
